tests/pytorch/test_float8tensor.py

```diff
@@ -302,14 +302,14 @@ def test_transpose(
         torch.testing.assert_close(x_fp8_t, x_t, **tols)
         assert not x_fp8._transpose_invalid, "Transpose cache reset incorrectly."
 
-        # Inplace update test.
+        # Inplace update test
+        x = x_fp8.from_float8() + 0.5
```

Comment:
What is the purpose of this line?

Reply:
It's the ground-truth value for the numerical check. I just pushed a fix so we actually use it (https://github.com/NVIDIA/TransformerEngine/pull/1127/commits/184540205160f1932896f0eb8e08509159c63f33).